src/screens/DiscoveryScreenV2/index.js

```diff
@@ -25,6 +25,13 @@ import {Header} from '../../components';
 import {colors} from '../../utils/colors';
 import {fonts} from '../../utils/fonts';
 import {withInteractionsManagedNoStatusBar} from '../../components/WithInteractionManaged';
+import {COLORS} from '../../utils/theme';
+
+const DiscoveryContainer = ({children}) => {
+  if (Platform.OS === 'ios') return <>{children}</>;
+
+  return <SafeAreaView style={{flex: 1}}>{children}</SafeAreaView>;
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `DiscoveryContainer` component has been moved from the bottom of the file to a higher position. This change is good for readability and maintainability as it's better to define components before they are used. However, please ensure that this component is not being referenced before its definition in any other part of the codebase to avoid potential ReferenceErrors.
```diff
+ import {COLORS} from '../../utils/theme';
+ 
+ const DiscoveryContainer = ({children}) => {
+   if (Platform.OS === 'ios') return <>{children}</>;
+ 
+   return <SafeAreaView style={{flex: 1}}>{children}</SafeAreaView>;
+ };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->